tests/test_homepage.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestHomePage():
+
+    @pytest.mark.nondestructive
+    def test_home_page(self, base_url, selenium):
```

Comment:
In the base.py file I've added the following method to verify if the login button is visible.
```
@property
def is_login_button_visible(self):
  return self.is_element_visible(self._login_locator)
```
It does the opposite of the `is_user_logged_in` method.  I left the check about the user not being logged in, in the test_home_page test, so it does the following:
```
assert not home_page.is_user_logged_in
assert home_page.is_login_button_visible
```
I'm not sure if both checks are need as they do similar things. Let me know if I should remove the is_user_logged_in check.